lib/vulcan.js

```diff
@@ -222,7 +222,8 @@ Vulcan.prototype = {
   isInsideTemplate: dom5.predicates.parentMatches(
       dom5.predicates.hasTagName('template')),
 
-  flatten: function flatten(tree, isMainDoc) {
+  flatten: function flatten(tree, rootDocPath) {
+    var isMainDoc = tree.href === rootDocPath;
```

Comment:
But perhaps the question is more about the behavior of the code as opposed to style.  Behavior is guaranteed because I pass in tree.href in the first instance.